### DISCUSSION TITLE
AIS Tropo

### DISCUSSION THREAD

**Discussion by Gazman429** [jvde-github/AIS-catcher#288] [Show and tell]
Had some good Tropo conditions a couple of months back, finally got around to make a short time lapse over a 4 day period, contacts received up to 2,000 km's away (1080 nmi's or 1242 miles).

https://github.com/user-attachments/assets/58f42055-c3d8-4afd-ace3-57c5c66cb6e6

**Comment by jvde-github** [comment]
Nice!
